treescript/tests/test_l10n.py

```diff
@@ -5,6 +5,10 @@
 import treescript.l10n as l10n
 
 
+async def noop_async(*args, **kwargs):
```

Comment:
good fodder for a shared test module :-)

Reply:
+1. The shared test module broke when i moved `test/` out from under `treescript/`. I'm now debating whether we should move back.